README.md

```diff
@@ -66,6 +66,7 @@ You can find the addon manifests and/or scripts under `${SNAP}/actions/`, with `
 - **storage**: Create a default storage class. This storage class makes use of the hostpath-provisioner pointing to a directory on the host. Persistent volumes are created under `${SNAP_COMMON}/default-storage`. Upon disabling this addon you will be asked if you want to delete the persistent volumes created.
 - **ingress**: Create an ingress controller.
 - **gpu**: Expose GPU(s) to microk8s by enabling the nvidia-docker runtime and nvidia-device-plugin-daemonset. Requires NVIDIA drivers to already be installed on the host system.
+- **istio**: Deploy the core Istio services. You can use the `microk8s.istioctl` command to mange your deployments.
```

Comment:
Which ones are the "core" services? What exactly is getting deployed when I enable this? Can we hyperlink "Istio" to the Istio website or GH repo for people who want to know more about it?

Reply:
I picked up "core" from here: https://istio.io/docs/setup/kubernetes/quick-start/#installation-steps right now I honestly do not know if there non-core services.